blueocean-core-js/package.json

```diff
@@ -49,15 +48,14 @@
     "mobx-utils": "1.1.2",
     "moment": "2.15.1",
     "pem-jwk": "1.5.1",
-    "react-remarkable": "1.1.1",
-    "react-router": "2.3.0"
-  },
-  "peerDependencies": {
-    "react": "^15.1.0",
-    "react-dom": "^15.1.0"
+    "react-remarkable": "1.1.1"
   },
   "devDependencies": {
+    "@jenkins-cd/design-language": "0.0.140-kz-4",
```

Comment:
@scherler it is, that's the whole point of this PR, JDL and core-js will no longer be published to npm, but to satisfy tools there will be a placeholder in the `devDependencies`